app/enterprise/1.5.x/kong-for-kubernetes/deployment-options.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Kong for Kubernetes deployment options
+---
+
+- [Deployment options](#deployment-options)
+- [DB-less versus database-backed deployments](#db-less-versus-database-backed-deployments)
+- [Choosing between DB-less or database-backed deployments](#choosing-between-db-less-or-database-backed-deployments)
+  * [Feature availability](#feature-availability)
+  * [Plugin compatibility](#plugin-compatibility)
+  * [Manual configuration](#manual-configuration)
+- [Migrating between deployment types](#migrating-between-deployment-types)
```

Comment:
You don't need this section, the docs now have a page ToC on the right-hand side. (I need to update the doc template though, thanks for the reminder)